templates/ntp.conf.freebsd.tmpl

```diff
@@ -40,6 +41,15 @@ pool {{pool}} iburst
 peer {{peer}}
 {% endfor %}
 
+#
+# If you want to pick yourself which country's public NTP server
```

Comment:
I don't understand why you'd want to add this comment in the template file (and therefore into the created ntp.conf file) - the "pools" cc_ntp configuration setting is used to specify pools to use other than the defaults, if someone wants to use country-specific ntp.org pool thenthey can specify it via the "pools" setting.

Reply:
I thought it might be good to have this file look the same as upstream.

but I think there's a good reason why i left this particular paragraph out initially